nemo/collections/tts/modules/fastspeech2_submodules.py

```diff
@@ -0,0 +1,347 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from nemo.collections.tts.helpers.helpers import get_mask_from_lengths
+
+from nemo.collections.tts.modules.transformer import PositionalEmbedding, TransformerLayer
+from nemo.utils import logging
+
+
+class FFTransformer(nn.Module):
```

Comment:
We should remove this file and reuse the classes from other TTS modules like fastpitch